crossbar/common/checkconfig.py

```diff
@@ -1544,6 +1544,29 @@ def check_listening_transport_web(transport, with_endpoint=True):
 _WEB_PATH_PATH = re.compile(_WEB_PATH_PAT_STR)
 
 
+def check_listening_transport_mqtt(transport):
```

Comment:
This is great .. so CB will be able to talk MQTT over Unix domain sockets;)

Default ports for MQTT are: "TCP/IP port 1883 is reserved with IANA for use with MQTT. TCP/IP port 8883 is also registered, for using MQTT over SSL." - But since we don't supply default ports for any of the other transports as well, it makes sense to require the user to explicitly specify the port to be used for MQTT as well. IOW: the MQTT ports should be simply documented in the CB docs, and that's it.

I am wondering if we could expand the "universal transport" also to allow running everything on 1 port (WebSocket/WAMP, RawSocket/WAMP, Web and MQTT - like on TLS/443) - of course that means MQTT clients need to connect on a non-standard port, but MQTT clients should allow that?

What is the first octet on an incoming MQTT connection?